test/basic.js

```diff
@@ -0,0 +1,106 @@
+var test = require('tape');
+var parse = require('../parse.js');
+var index = require('../index.js')
+
+/* BEGIN parse.js tests */
+
+test('parseResource playlist test 1', function (t) {
```

Comment:
Test names should be statements that the test asserts. See below for some examples.